sources/LLVMSharp.Interop/Extensions/LLVMAttributeRef.cs

```diff
@@ -8,9 +8,68 @@ public unsafe partial struct LLVMAttributeRef(IntPtr handle) : IEquatable<LLVMAt
 {
     public IntPtr Handle = handle;
 
-    public readonly uint Kind => LLVM.GetEnumAttributeKind(this);
+    /// <summary>
+    /// <see href="https://github.com/llvm/llvm-project/blob/a64e6f49284a7ffd5401183d0e94a94a7de39cfb/llvm/include/llvm/IR/Attributes.h#L233"/>
+    /// </summary>
+    public readonly bool HasKindAsEnum => Handle != IntPtr.Zero && LLVM.IsStringAttribute(this) == 0;
```

Comment:
nit: repo convention is to parenthesize comparisons on either side of an `&&` or `||`
```suggestion
    public readonly bool HasKindAsEnum => (Handle != IntPtr.Zero) && (LLVM.IsStringAttribute(this) == 0);
```

This helps avoid ambiguity when reading the code as different languages can have varying precedence rules and it helps isolate the two conditions being short circuited.